src/main/java/de/cuuky/varo/game/start/ProtectionTime.java

```diff
@@ -31,33 +37,36 @@ public void run() {
 					return;
 				}
 
-				if (this.protectionTimer == 0) {
+				if (ProtectionTime.this.protectionTimer == 0) {
 					Main.getLanguageManager().broadcastMessage(ConfigMessages.PROTECTION_TIME_OVER);
 					Main.getVaroGame().setProtection(null);
 					scheduler.cancel();
-				} else if (protectionTimer % ConfigSetting.STARTPERIOD_PROTECTIONTIME_BROADCAST_INTERVAL.getValueAsInt() == 0 && this.protectionTimer != timer)
-					Main.getLanguageManager().broadcastMessage(ConfigMessages.PROTECTION_TIME_UPDATE).replace("%minutes%", getCountdownMin(protectionTimer)).replace("%seconds%", getCountdownSec(protectionTimer));
+				} else if (ProtectionTime.this.protectionTimer % ConfigSetting.STARTPERIOD_PROTECTIONTIME_BROADCAST_INTERVAL.getValueAsInt() == 0) {
+					Main.getLanguageManager().broadcastMessage(ConfigMessages.PROTECTION_TIME_UPDATE)
+						.replace("%minutes%", getCountdownMin(ProtectionTime.this.protectionTimer))
+						.replace("%seconds%", getCountdownSec(ProtectionTime.this.protectionTimer));
+				}
 
-				this.protectionTimer--;
+				ProtectionTime.this.protectionTimer--;
 			}
 		}.runTaskTimer(Main.getInstance(), 1L, 20L);
 	}
 
-	private String getCountdownMin(int sec) {
+	public String getCountdownMin(int sec) {
 		int min = sec / 60;
-
-		if (min < 10)
-			return "0" + min;
-		else
-			return min + "";
+		return (min < 10) ? "0" + min : String.valueOf(min);
 	}
 
-	private String getCountdownSec(int sec) {
+	public String getCountdownSec(int sec) {
 		sec = sec % 60;
+		return (sec < 10) ? "0" + sec : String.valueOf(sec);
+	}
 
-		if (sec < 10)
-			return "0" + sec;
-		else
-			return sec + "";
+	/**
+	*
+	* @return the protection timer or null if no protection time is set
+	*/
+	public int getProtectionTimer() {
+		return this.protectionTimer > 0 ? protectionTimer : null;

```

Comment:
```suggestion
		return this.protectionTimer;

```
`int` can not be `null`